readme.md

```diff
@@ -2,7 +2,7 @@
 
 ## Site
 
-Hosted using github pages [here](https://pages.github.com/)
+A demo is hosted using github pages [here](https://alexodan.github.io/navbar-challenge/)
```

Comment:
tip:  [avoid click here links](https://uxmovement.com/content/why-your-links-should-never-say-click-here/)

```suggestion
Checkout the [demo page](https://alexodan.github.io/navbar-challenge/).
```

(saying where it's hosted it's not relevant. Seeing a shiny link saying "demo" is the juicy part